.github/workflows/yaml2json.yml

```diff
@@ -0,0 +1,51 @@
+name: YAML2JSON
+
+# run this on push to master
+on:
+  push:
+    branches:    
+      - master
+      - try-github-actions
```

Comment:
Is that still needed? :)

Reply:
probably not, going to remove that before merging this PR. But it will run the action if I make adjustments to this branch so I'd keep it for testing until it is ready to merge.